### ISSUE TITLE
Monthly Mycetoma Meeting 23rd January 2024

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Monthly-zoom-meetings#15]
Time: Jan 23rd (8 am UK) [Timezones](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=1&day=23&hour=8&min=0&sec=0&p1=43&p2=136&p3=37&p4=771&p5=235&p6=152&p7=770&p8=250&p9=196&p10=240).

The Outlook invite can be found [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/2#issuecomment-1905476888)

Location: today we will meet at https://ucl.zoom.us/j/99789829423 and in case of double failure use [MycetOS meeting teams link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2I2MDcxZTgtYjZjOS00YTdmLTgwZTMtNWQ3OTc4NDY0MDNj%40thread.v2/0?context=%7b%22Tid%22%3a%22526638ba-6af3-4b0f-a532-a1a511f4ac80%22%2c%22Oid%22%3a%22a5ed84b8-5af1-42c6-9f8d-240f23a5d997%22%7d)
Chair: 
Minute taker: ErasmusMC
Previous meeting: #14 
Recording: [here]
Present:
Apologies: 

**AGENDA**

@OpenSourceMycetoma/corecontrib please flag up any key issues ahead of the meeting, and please note any apologies above.

### **In vitro** screening**

- [ ] @MA-Jjingyi @marijschippers to report on any _in vitro_ or _in vivo_ screening that has taken place since the last meeting

### **Series 1 - Fenarimols**
- [ ] Final Molecules/SAR Gaps (summarised [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1530752314). Latest status was described in the last meeting (#14). In summary:

- [ ] @dosreic to investigate whether it is possible to have SCYX0001294212 EPL-BS0121 and SCYX0001315839 EPL-BS0143 shipped from TCG to Wendy.

- [x] @fantasy121 to add the [five compounds tested](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/12#issuecomment-1828164980) to the Master List and generate MyOS codes and send those codes to @MA-Jjingyi and/or @marijschippers to ensure data can be properly reported.

- [ ] @MA-Jjingyi to advise whether more data coming on these compounds

**Fenarimol Paper**

Most recent file version on Dropbox remains: _Fenarimol Paper October 3rd V2.2_ edited last Jan 22nd 2024.

- [ ] @fantasy121 and @dmitrij176 completed all assignments to them on experimental?
- [ ] @fantasy121 and @dmitrij176 What is known of related structures inhibiting CYP51, i.e. how do our results compare to what is known of Cyp51 inhibition?
- [ ] Compound numbering: use MyOS codes, but can use other numbers for compounds without those codes.

Highlighting recent hits in this series: from @MA-Jjingyi latest data, it would be helpful to have a diagram with structures. MYOS00510 and MYOS00518 were hits? Do recently-tested compounds need to be incorporated in the fenarimol manuscript? New SAR diagrams ideally need to be added to the wiki with codes, or we risk missing key insights from the new data.

Examples (are there others?):
[Data from Sept 21](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/files/12684110/26092023.MyctOS.meeting.-MA.pptx)

[ ] Modelling: @meh-cyprian to proceed to try to model fenarimol compounds in potential protein target (described [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/68#issuecomment-1546869698) or to clarify what more data are needed. Aim is to better understand the SAR via docking vs homology model of protein. @gemmaturon and @miquelduranfrigola presented their work that was done when visiting @meh-cyprian. Were there actions arising from that presentation?

### **Siderophores:**
Fluoroscent siderophore was tested in the larvae in the last weeks of the season.

- [x] @wwjvdsande to process the data to correct for autofluorescence. In the absence of this being on a wiki, maintain link with [this issue](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/issues/10). <-- what were the outcomes of this testing?

### **Series 2 - Aminothiazoles**
- [ ] Update from @KlementineJBS from the [schools meeting](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/17)? --> discussion was on which molecules where going to be made next.

Molecules made by UCL MSc cohort (23 entries) were shipped to Erasmus MC for testing. Also Whitgift compound (MYOS_0590_00_01).

- [x] @MA-Jjingyi to report back on these data, potentially with update from Erika Loizidou (UCL). Note that [the current SAR page](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki/Structure-Activity-Relationships) is out of date and [this one](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki/Overall-summary) should be used? (Great job keeping this updated @KlementineJBS !)

@KlementineJBS wrote a guide on the [Wiki](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki) on how to share compounds in a way that will hopefully prevent duplication of both compounds themselves and MYOS codes. Also posted some compounds to be made next [here](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/19).

- [ ] @AndyBishopSevenoaks @JuliaSarju are there any samples likely to be incoming? <-- update from @P-Rayner was posted [here](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/23) and @KlementineJBS was in touch with @P-Rayner about shipping etc.
- [ ] @dosreic to try to secure samples related to this paper https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/11 ?

**Series 5 - Benzimidazoles**
Good potencies were observed for recent compound batches of benzoxazoles (e.g. [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/82#issuecomment-1325150305)). These resemble the benzimidazoles which have shown potent in vivo efficacy.

Testing is ongoing. So far MYOS00527 was the only hit.

- [x] @MA-Jjingyi any other data from this series?

Some oxazoles will be tested against _Falciformispora senegalensis_ too (by whom?). This series will be evaluated _in vivo_ in the new larval season <-- is this still to be done?

More benzoxazoles: @AndreaOtago says that the CHEM 305 students will be making more benzoxazoles in April 2024 as part of their lab project. @dosreic suggested doing some PK 'SAR' follow up analogues so this could inform both DNDi's project and this project. @dosreic to follow up the DMPK results.

- [ ] @AndreaOtago and @dosreic to update on planned designs and whether other students could make additional compounds using the same approach?

- [ ] @memm0740 to search for whether additional samples could be obtained from other labs. <-- can be closed, though still needed!
- [ ] @memm0740 to update on progress of synthesis. Done [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/7#issuecomment-1622784020) <-- Can this be added to the wiki by a member of the Sydney team, so we can close this action item?
- [ ] @memm0740 to post link to raw data on ELN <-- can this please be put on the wiki by a member of the Sydney team?

### **Series 6: ketoximes**
Latest compound shipment https://github.com/OpenSourceMycetoma/Series-6-Ketoximes/issues/5, to be tested by @MA-Jjingyi.

- [ ] @dmitrij176 to summarise latest series data in a diagram and compare with previous most-relevant data to reveal what has been learned.

@bebi78 had suggested evaluating different salts of the ketoximes, still of value for future. Perhaps this can go on the wiki under future plans? <-- outdated. Would be useful to capture on wiki and then delete from here.

### **Series 3: Tricyclics/Phenothiazines**
- [x] NEU-Curaxin compounds have been given MyOS codes and can be tested - @wwjvdsande any ETA on data for this? needs to be pushed to master list and shared with @lferrins and @P-Rayner <------> **Wendy said a week or two is needed before data is released.**
- [x] @JuliaSarju and team to give update on phenothiazine synthesis projects - https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/issues/8 

### **Global Health Priority Box***

This has been tested and reported by @MA-Jjingyi [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/12#issuecomment-1729421998). Three compounds were taken on to _in vivo_ testing. Outcome is some promise for compound MMV1804559.

- [x] Should MMV1804559 be pursued further?

### **Kinase inhibitor library.** 
The KCGS library was screened. Additional drugs of similar classes were sent by @bebi78 and screened by @marijschippers. In total 6 drugs from the KCGS appeared to be hits along with one from the package sent by @bebi78. 

- [x] Data to be reported and uploaded.

### **Bioinformatics**
- [ ] @osmanwa to generate a phylogenetic tree to work out relationships of various species to allow for potential repurposing of hits vs closely-related organisms. @TariqAyub to contribute to this?

### **Plant Extracts from Nigeria**

Further to the conversation [here](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/issues/2) we need to establish

- [ ] Whether the extracts have been evaluated @wwjvdsande @MA-Jjingyi 

- [ ] How we assign codes to these samples @bebi78 

- [ ] Whether more samples are incoming and how we can obtain costs for shipment @nwezemeka @dosreic 

### **Misc updates**:

- [x] Jan 30th is World NTD Day - how should we mark this to spread the word? Amplification of socmed activity by @dosreic would be very good.

- [ ] @mattodd has received DMPK measurements by Dundee. Compound selection took place in [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/8) - @dmitrij176 to post there the chemdraw and spreadsheet of the final list of compounds that was sent to Dundee so we can correlate data with structure.

- [ ] @memm0740 to enter the data for [Rohan Davies' compounds](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/84#issuecomment-1507518700) into the Master List. They need [MyOS codes](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention). <-- done? Can this be closed?

- [ ] @MA-Jjingyi previously presented larval toxicity studies results from recent compound screening. Data were presented [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/12#issuecomment-1729421998), but how do we fold the data into the Master List and the various wikis? @KlementineJBS gave a summary [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/12#issuecomment-1746122774). Do we have a strategy for toxicity measurements more generally?

### **Next Meetings**
Feb 27th noon UK time ([Timezones](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=2&day=27&hour=12&min=0&sec=0&p1=43&p2=136&p3=37&p4=771&p5=235&p6=152&p7=770&p8=250&p9=196&p10=240))

**Comment by mattodd** [comment]
@P-Rayner - it may be that nobody invited you to this meeting, sorry if so. If you can make it, great, otherwise thanks for the update at and do please keep us posted on any developments.

**Comment by fantasy121** [comment]
@MA-Jjingyi @marijschippers I've given the five Epichem compounds recently screened MYOS codes in the Master List. More evaluation data can now be added.
![Screenshot 2024-01-23 at 5 29 09 pm](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/assets/36176821/3a05fb91-16f2-471b-883b-3ab21f61fd93)

**Comment by miquelduranfrigola** [comment]
Hi @mattodd there were no actionables on our end from the previous meeting

**Comment by mattodd** [comment]
Update slide from Erika for compounds made by MSc class at UCL.
[Erika MSc group project_aminothiazoles_summary.pptx](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/files/14020093/Erika.MSc.group.project_aminothiazoles_summary.pptx)

**Comment by MFernflower** [comment]
Structure of MMV1804559

![image](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/assets/3164942/f4fe6e73-e61e-405b-a1b5-a956856e2074)

**Comment by AndreaOtago** [comment]
As mentioned in the meeting, all of the 'series 5' benzoxazoles, the 'UOxxx' codes in the 'other ID' column, have been tested by DNDi partners (tagging @dosreic) against a parasite panel and also for cell cytotoxicity. This data is publicly available at https://www.dropbox.com/s/0ex9pv43c1mmazo/OSN%20P5%20Data%20set.sdf?dl=0 (SDF file) or https://www.dropbox.com/s/l67p8xa3raa1nrj/OSN%20P5%20Data%20set.xlsx?dl=0 (excel file) from this website https://dndi.org/research-development/portfolio/open-synthesis-network/ (ie OSN project 5). There are around 100 benzoxazoles of similar structure in this data. Taking a quick look it looks like one your hits, MYOS_00527_00_01 aka DNDI0004042515 showed cytotoxicity.

**Comment by fantasy121** [comment]
Sorry I missed the meeting as I made a mistake with the time (thought it was later).

I'm currently putting together the concluding remarks following the SAR, in the manuscript (the item "What is known of related structures inhibiting CYP51, i.e. how do our results compare to what is known of Cyp51 inhibition?". This will be done by tomorrow on my part.

I revisited the SAR section and because my final batch MYOS510–520 has been screened, more SAR trend can be seen. I've just written this section for my thesis and I think it would be valuable to add this to the manuscript. Particularly of value is match-pairing grid of 24 compounds with variation of substitution patterns (Ring 2) vs changes to Ring 3 aromaticity (first photo below) @mattodd okay to do this?
![Screenshot 2024-01-23 at 8 31 58 pm](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/assets/36176821/f424b04b-a76f-483a-9648-a2799a370aa1)
![Screenshot 2024-01-23 at 8 37 21 pm](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/assets/36176821/30968c8e-a230-4767-a20c-d0aa6bae1902).

If these are to be added, I can also update the manuscript SI excel file with these entries as well. @wwjvdsande checking with you that this is okay to do.

**Comment by mattodd** [comment]
@fantasy121 Yes, sounds great, please add. If you want, you can add this in as a new scheme in the right place and include any conclusions you've derived and then we can all look at how bst to merge this into the main MS? Let us know when you're done, and we can then take a look. Ditto the MOA remarks - please add in what you have and then we can take a look. Ultimately these (updated SAR, comparison with lit MOA) are the final two parts needed for the manuscript.

**Comment by P-Rayner** [comment]
> @P-Rayner - it may be that nobody invited you to this meeting, sorry if so. If you can make it, great, otherwise thanks for the update at and do please keep us posted on any developments.

@mattodd many thanks for the invite. Sorry I didn't pick it up in time but I will try to be at the next one. We have sent off our samples, so hopefully they will arrive soon (custom problems depending!).

**Comment by MFernflower** [comment]
I will make myos code for the York series 3 molecules in a few hours
@mattodd

**Comment by KlementineJBS** [comment]
# Minutes 

As a document:

Relevant files: 
[Erika.MSc.group.project_aminothiazoles_summary.pptx](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/files/14033576/Erika.MSc.group.project_aminothiazoles_summary.pptx)

Chair: ErasmusMC
Minute taker: USyd
Previous meeting:  
Recording: [here]
Present: [@mattodd](https://github.com/mattodd) @MFernflower @wwjvdsande @dmitrij176 @KlementineJBS, @marijschippers Mickey, @AndreaOtago 
Apologies: @fantasy121 [@MA-Jjingyi](https://github.com/MA-Jjingyi)
## *In vitro* screening

*Report from Wendy at ErasmusMC*
-	The results of the latest screening have been added to the Masterlist. A new tab has been created for a new screen of compound. 
- Wendy and team were not able to find the plant extract samples from Nigeria – can @bebi78 or [@nwezemeka](https://github.com/nwezemeka) advise what these compounds look like? See relevant issue here. 
- NEU compounds (phenothiazines) are being re-screened due to some discrepancies in results. New results should be available in 1-2 weeks. 
- One compound, from the Global Health Priority Box, had quite promising results: it both prolonged larval survival and induced morphological changes in the grains (decreasing their size). Could potentially be the start of a new series – chemists should have a look at the structure and consider. 
- Results from UCL are slightly confusing – some duplicate compounds have had different results when tested twice. See Erika’s slides for details: 

### Notes on screening 
-	Imperative that all compounds shipped to Erasmus have clear MYOS codes or at very least are labelled as MycetOS compounds (with an institutional identifier). 
-	we need to make sure we can capture/keep track of SAR. Will trial a colour code system, where the biology team shades recently tested samples in yellow in the masterlist. Once a chemist has digested this data in some way, e.g. as in Series 2, the chemist should remove the yellow shading from the masterlist. 


## **Series 1 - Fenarimols**

- Final Molecules/SAR Gaps summarised [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1530752314). In summary: 

- No update on @dosreic to investigating whether it is possible to have SCYX0001294212 EPL-BS0121 and SCYX0001315839 EPL-BS0143 shipped from TCG to Wendy.

- @fantasy121 has added [five compounds tested](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/12#issuecomment-1828164980) to the Master List 
-	Dmitrij has done some modelling for both fenarimols and ketoximes (more on ketoximes later). There is some evidence to suggest a correlation between binding in hydrophobic pocket / spatial arrangement and activity of compounds. Currently, the literature about this is very disperse and unspecific.
-	Should keep in mind that compounds which have good docking/modelling results but aren’t active in vitro are possibly not able to cross the cell wall or membrane. 
-	Currently assume that both ketoximes and fenarimols are CYP451 inhibitors. 
-	NDI has phosphoconazole (name incorrect?), a prodrug for ravuconazole, in trials right now. It inhibits CYP451 like itraconazole, the currently used medication. 


### Fenarimol paper

- Most recent file version on Dropbox remains: _Fenarimol Paper October 3rd V2.2_ edited last Jan 22nd 2024.
-	Dmitrij has updated all his compounds bar some characterisation
-	Hung is still updating his compounds and will fold new data from MYOS510-510 into SAR, with conclusions from this work 
-	Once this is done, Mat and rest of the team can consider changes
-	Again, need to make sure SAR is being digested/reviewed regularly – should create diagram with structures of recent testing.

## **Siderophores:**

-	Fluorescent siderophore was tested in the larvae in the last weeks of the season. Results were not good, seemingly because the siderophores were not internalised (couldn’t cross cell wall or membrane). 
-	Siderophores were derived from Aspergillus, so we probably need to try using siderophores from the actual Madurella mycetomatis instead. 
-	On hold until siderophores can be sourced. 

## **Series 2 - Aminothiazoles**

-	UCL compounds were not very successful. This may be due to the lack of pyridine ring on the left-hand (bromoketone-derived) side. 
-	Most potent compounds so far are those with halogen substitutions on right-hand ring. Should look into making doubly-substituted halogen compounds in further work. 
-	Still have little insight into MOA, though results from this halogen substitution indicate that EWGs play a role
-	Should try testing with itraconazole to see if there is a synergistic effect – if there is, at least we’ll know that the aminothioazoles have a different MOA (i.e. do not work by inhibiting CYP451)
-	Desiree from Whitgift School is in communication with UCL about getting compounds purified and analysed. @KlementineJBS has provided some target molecules for Whitgift to work on this year. 
-	No update from @AndyBishopSevenoaks or @dosreic relating to [this paper]( https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/11)
-	@JuliaSarju and @P-Rayner have sent through some compounds from Uni of York which should arrive at Erasmus soon, see [here](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/23).  The aminothiazoles have MYOS codes already. 

## Series 5 - Benzimidazoles

-	Usyd is no longer actively making benzimidazoles 
-	Good potencies were observed for recent compound batches of benzoxazoles (e.g. [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/82#issuecomment-1325150305)). These resemble the benzimidazoles which have shown potent in vivo efficacy.
-	MYOS00527 is the only hit from the most recent batch, but testing is ongoing. They are screened against C elegansis and falciforme. This molecule is from a batch made for a different DNDI project for leishmaniasis – it was low potency in those screens. It also showed signs of cytotoxicity in previous screening.
-	The benzoxazoles from Otago have a full panel of screening available online [here]( https://dndi.org/research-development/portfolio/open-synthesis-network/) or in the dropbox as an [SDF](https://www.dropbox.com/s/0ex9pv43c1mmazo/OSN%20P5%20Data%20set.sdf?dl=0) or [excel](https://www.dropbox.com/s/l67p8xa3raa1nrj/OSN%20P5%20Data%20set.xlsx?dl=0). 
-	Three of the benzoxazole compounds, UO17, UO22 and UO24, are eligible for *in vivo* screening later in the year. These screens test efficacy first and then toxicity.
-	Students at Otago will be making more benzoxazoles in April 2024 as part of their lab project. @dosreic suggested doing some PK 'SAR' follow up analogues so this could inform both DNDi's project and this project. @dosreic to follow up the DMPK results.
-	Still need someone to find out whether additional samples could be obtained from other labs. 
-	@KlementineJBS will add an update on @memm0740 ‘s synthesis and raw data from ELN to the Wiki, for now see [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/7#issuecomment-1622784020)

## **Series 6: ketoximes**

- latest compound shipment https://github.com/OpenSourceMycetoma/Series-6-Ketoximes/issues/5, to be tested by @MA-Jjingyi.
-	Oxiconazole nitrate (Z isomer) has been the most potent so far. It was purchased commercially from Sigma Aldrich. Should note that it does not have a pyridine ring like other compounds tested. 
-	Compound MYOS00419 has not been tested - @wwjvdsande to check on this 
-	As mentioned, some correlations found between docking results and potency. E.g. E isomer of DM88-6 is not active, while Z isomer is. 
-	Di-fluoro substitutions on pyrifenox had high potency but still lower than pyrifenox 
-	Doubly-alkylated compounds had lower potency. Removing pyridine ring, adding halogens onto second benzyl ring or substituting with bromine all reduced potency as well. 
-	Some issues with degradation of pyridinium salts – e.g. DM96-1 has low potency, but degrades into a different molecule with higher potency. 
-	Dmitrij to make and share diagram summarising these findings
-	@bebi78’s suggestion to evaluate different salts of ketoximes should be moved onto Wiki as future work 


## **Series 3: Tricyclics/Phenothiazines**

-	As discussed, NEU compounds are being rescreened 
-	@P-Rayner has sent compounds from Uni of York (see [here](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/issues/8)). @MFernFlower volunteered to give these phenothiazines MYOS codes

## **Bioinformatics**

No one on the call could update on Bioinformatics. 

## **Misc updates**:

-	Should amplify message on World NTD Day (Jan 30th). @mattodd to share something via Twitter, other Twitter users should retweet this.
-	DNDI will presumably be sharing things as well
-	ErasmusMC will have a press release 
-	Others should consider asking their institution to tweet/do a press release etc
-	Have created an issue for suggestions on any other methods
-	@mattodd and @dmitrij176 will review DMPK measurements by Dundee and report back next meeting. Compound selection took place in [here](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/8) 
-	@KlementineJBS will take over data for [Rohan Davies' compounds](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/84#issuecomment-1507518700) into the Master List. They need [MyOS codes](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention). 
- should add an extra column to the masterlist about screening compounds for toxicity. Can also consider other ways to capture this data e.g. in SAR diagrams.

**Comment by wwjvdsande** [comment]
@P-Rayner the compounds arrived today. We will dilute them and start the screening process. On the bottle it was not written down how much mg was in each bottle. Do you know or do we need to weigh?

**Comment by P-Rayner** [comment]
Hi @wwjvdsande I'm pleased the compounds have arrived - I kept getting emails to say they were delayed due to the storm. 
Apologies - I didn't record how many mg in each bottle - I will do for next time!

**Comment by mattodd** [comment]
I wonder - for World NTD day on Tue could we highlight:

1) Any molecule from the recent screening that was unusually potent?
2) The molecule that appeared to modify the grain morphology?

Apologies, because I had to leave early, I don't know if there was something for 1) and I don't know which molecule was 2). If someone could point me in the right direction I can try to highlight on Tue.

**Comment by fantasy121** [comment]
> @fantasy121 Yes, sounds great, please add. If you want, you can add this in as a new scheme in the right place and include any conclusions you've derived and then we can all look at how bst to merge this into the main MS? Let us know when you're done, and we can then take a look. Ditto the MOA remarks - please add in what you have and then we can take a look. Ultimately these (updated SAR, comparison with lit MOA) are the final two parts needed for the manuscript.

@mattodd I've added the additional SAR information into the manuscript. Updated MS version to _Fenarimol Paper January 29 V2.3_ as current.

I've added the new compounds in the Expo details section but am working on some final characterisations.

**Comment by wwjvdsande** [comment]
@mattodd the molecule which came out of the latest screenings from the MMV boxes and which resulted in smaller grains was: MMV1804559

**Comment by MFernflower** [comment]
@wwjvdsande Just uploaded the structure png

**Comment by mattodd** [comment]
Hi @MFernflower would you be willing to do a search for commercial sources of MMV1804559 and close analogs? Smaller grain size is a potentially significant finding.

@wwjvdsande is it possible to check whether this compound alters the penetration into grains e.g. by co-administration with a compound that we already know to be partially effective in the in vivo model?

**Comment by wwjvdsande** [comment]
@mattodd that is a difficult question. Amphotericin B does work well in the larvae but we don't know yet what MMV1804559 exactly does. We are looking in literature to find some clues. We could try to combine MMV1804559 and amphotericin B but this would not automatically mean that it will allow better penetration. We did not see synergy between itraconazole and MMV1804559.

**Comment by mattodd** [comment]
This is very interesting.
Do you have enough of this sample for now?
Does the grain size depend on the concentration of the molecule? (That may be a tough question to answer)
Besides size, I wonder if there is anything different about the grain morphology that is measurable?

**Comment by wwjvdsande** [comment]
@mattodd unfortunately we only test at standard concentrations. We had to lower the concentrations due to toxicity issues but at a lower concentration it did work. We looked with histology at the grain morphology but so far no clear differences there, only the size.

**Comment by MFernflower** [comment]
@mattodd one isomer of MMV1804559 gets a hit on pubchem and a related paper

https://pubchem.ncbi.nlm.nih.gov/compound/44528300

https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0149996

**Comment by MFernflower** [comment]
@mattodd

One isomer of the MMV compound has a chembl entry as CHEMBL527240

seems to have been tested in a kala-azar screen?

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib I uploaded some new data in the spreadsheet. Please take a look at it. It includes the data from @dmitrij176 s compounds as well as the NEU compounds as requested by @MFernflower

**Comment by dmitrij176** [comment]
Thank you @wwjvdsande
